tests/functional/test_download.py

```diff
@@ -1,9 +1,20 @@
 import os.path
 import shutil
+import sys
 import textwrap
 from hashlib import sha256
 
 import pytest
+from pip._vendor.packaging.tags import \
```

Comment:
nit:

    from ... import (
        ...
    )

Reply:
I wrote the version with the parentheses, so I don't know where the backslash escapes came from. Maybe one of the autoformatters doesn't like the end of line comments?

Reply:
isort does it. I suggest we leave it and if we care enough we should determine how to do it automatically later.

Reply:
Looking at https://github.com/timothycrosley/isort/blob/623be1efff73d17219052d1afb032fd2bedb30c4/isort/output.py#L390, it's almost certainly the trailing comments that are triggering the single line output.

Adding `isort:skip` to prevent that was straightforward enough, so I went ahead and changed the formatting back to the parenthesized form.